packages/block-library/src/form/index.php

```diff
@@ -96,7 +96,7 @@ function block_core_form_send_email() {
 		'<a href="' . esc_url( get_site_url( null, $params['_wp_http_referer'] ) ) . '">' . get_bloginfo( 'name' ) . '</a>'
 	);
 
-	$skip_fields = array( 'formAction', '_ajax_nonce', 'action' );
+	$skip_fields = array( 'formAction', '_ajax_nonce', 'action', '_wp_http_referer' );
```

Comment:
Maybe too late but it'd be great if we could instead of keeping alist of fields to skip, find a way to identify what to include - like if we had a prefix for all fields that should be emailed or something?

Reply:
Well, the args that need to be ignored are finite and known... That's why I thought it best to keep it clean and not add prefixes to the other ones. But if you think it's better the other way around, then sure.. we can do that 😃

Reply:
> the args that need to be ignored are finite and known

Today! But one day some hidden thing _don't_email_me will show up and be emailed :)